elliott/elliottlib/metadata.py

```diff
@@ -224,9 +224,16 @@ def get_latest_build(self, default: Optional[Any] = -1, assembly: Optional[str]
 
             el_ver = None
             if el_target:
-                el_ver = isolate_el_version_in_brew_tag(el_target)
+                if isinstance(el_target, int):
```

Comment:
I'd pull these checks into a more abstract utility function for finding el. e.g. `isolate_el_version` which can take strings, ints, and tag names.